SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -2273,7 +2276,7 @@ void CDriver::Iteration_Preprocessing(CConfig* config, CIteration *&iteration) c
   if (rank == MASTER_NODE)
     cout << endl <<"------------------- Iteration Preprocessing ( Zone " << config->GetiZone() <<" ) ------------------" << endl;
 
-  iteration = CIterationFactory::CreateIteration(static_cast<ENUM_MAIN_SOLVER>(config->GetKind_Solver()), config);
+  iteration = CIterationFactory::CreateIteration(static_cast<MAIN_SOLVER>(config->GetKind_Solver()), config);
```

Comment:
These static casts can be removed now, do a search & replace for others please.
```suggestion
  iteration = CIterationFactory::CreateIteration(config->GetKind_Solver(), config);
```